mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/ValidateUtil.kt

```diff
@@ -0,0 +1,12 @@
+package org.mifos.mobilewallet.mifospay.utils
+
+import android.text.TextUtils
+import android.util.Patterns
+
+object ValidateUtil {
+  fun isValidEmail(email: String?): Boolean {
+    return (!TextUtils.isEmpty(email)
```

Comment:
As far as I know, you can convert a function into compact and omit the return keyword. What do you think?